aspnet/SameSite/system-web-samesite.md

```diff
@@ -31,6 +31,8 @@ SameSite support was first implemented in .NET 4.7.2 using the [2016 draft stand
 
 The November 19, 2019 updates for Windows updated .NET 4.7.2+ from the 2016 standard to the 2019 standard. Additional updates are forthcoming for other versions of Windows. See the following KB's for more information:
 
+[!INCLUDE[](~/includes/kbs-samesite.md)]
```

Comment:
Rather than rendering the table in each doc, can the table be its own page that gets linked to?

Reply:
Good idea. I'll do that.